index.html

```diff
@@ -0,0 +1,857 @@
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" href="./css/style.css">
+  <title>TNW</title>
+</head>
+<body>
+  <header>
+    
+    <div class="black-header">
+      <nav class="black-header-left">
+        <ul>
+          <li class="black-header-left-new">News</li>
+          <li>Events</li>
+          <li>Business</li>
+          <li>AMAs</li>
+          <li>Spaces</li>
+          <li class="black-header-left-last">Terms & Conditions</li>
+        </ul>
+      </nav>
+
+      <nav class="black-header-right">
+        <ul>
+          <li>About</li>
+          <li>Advertise</li>
+          <li>Jobs</li>
+          <li>Contact</li>
+        </ul>
+      </nav>
+    </div>
+
+      
+    
+
+  </header>
+  
+  <div class="white-header">
+    <div class="white-header-left">
+      <div class="header-logo">
+        <div class="space-holder"></div>
+        <img src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg" alt="logo of Spider chicken" >
+      </div>
+      
+      <ul>
+        <li>LATEST</li>
+        <li>HARD FORK</li>
+        <li>PLUGGED</li>
+        <li>README</li>
+        <li>GROWTH QUARTERS</li>
+        <li>NEURAL</li>
+      </ul>
+    </div>
+
+    <div class="white-header-right">
+      <div class="search-container">
+        <i class="fa fa-search search"></i>
+      </div>
+
+      <div class="reorder-container">
+        <i class="fa fa-reorder reorder"></i>
+      </div>
+      
+    </div>
+  </div>
+  
+  <main>
+    <section class="titles"> 
+        <div class="title1">
+          <a href="#" class="lower-left">
+            <div class="title-logo-div">
+              <img src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg" alt="TNW-Logo">
+              <p>Tuesday — July 28, 2020</p>
+            </div>
+              <h2>TECH</h2>
+              <h1>How to write cleaner code with Javascript</h1>
+              <p>by Live Code Stream  <i class="fa fa-clock-o"></i> 1 hour ago <i class="fa fa-share-alt"></i> 0</p>
+          </a>
+        </div>
+     <div class="title2">
+      <a href="#" class="lower-left">
+          <h2>TECH</h2>
+          <h1>Remote jury trials in times of COVID-19</h1>
+          <p>by The Conversation <i class="fa fa-clock-o"></i> 4 hours ago <i class="fa fa-share-alt"></i> 0</p>
+      </a>
+    </div>
+    <div class="title3">
+      <a href="#" class="lower-left">
+          <h2>TECH</h2>
+          <h1>How Adobe Photoshop is used in the search for missing children</h1>
+          <p>by Rachel Kaser <i class="fa fa-clock-o"></i> 11 hours ago <i class="fa fa-share-alt"></i> 9</p>
+      </a>
+    </div>
+      
+      
+      
+    </section>
+    <a href="#"><h2 class="section-heading">Latest news <img src="./images/next-page.png" alt="next-page"></h2></a>
```

Comment:
from this line and until the main closing tag, wrap everything inside a div that needs to be centered and have a fixed width:

| *Clone* | *Original* |
|:---:|:---:|
| ![image](https://user-images.githubusercontent.com/34653764/88865594-350b7e00-d200-11ea-9d95-d6311341c7f5.png) | ![image](https://user-images.githubusercontent.com/34653764/88865574-2ae97f80-d200-11ea-88ad-8e81ddf9f082.png) |